pkg/services/accesscontrol/resourcepermissions/api.go

```diff
@@ -35,13 +35,13 @@ func (a *api) registerEndpoints() {
 	uidSolver := solveUID(a.service.options.UidSolver)
 	disable := middleware.Disable(a.ac.IsDisabled())
 	a.router.Group(fmt.Sprintf("/api/access-control/%s", a.service.options.Resource), func(r routing.RouteRegister) {
-		idScope := accesscontrol.Scope(a.service.options.Resource, "id", accesscontrol.Parameter(":resourceID"))
+		scope := accesscontrol.Scope(a.service.options.Resource, a.service.options.ResourceAttribute, accesscontrol.Parameter(":resourceID"))
```

Comment:
nit: again, should `resourceID` be renamed to something else? I guess it's more tricky here, because using both IDs and UIDs means that endpoints for some resources will be using IDs and for others UIDs 🤔 

I guess it's fine, but our docs will need to be really clear about this, if we do document these endpoints.